usagov/vote-gov-drupal
# VOTE-3211 Update state title to use display content value

## Jira ticket

https://cm-jira.usa.gov/browse/VOTE-3211

## Description

Update state title to use display content value

## Deployment and testing

### Post-deploy steps

1. `lando drush cr`

### QA/Testing instructions

1. visit http://vote-gov.lndo.site/admin/content/block/23 and change the Page display title to  `Register NOW to vote @state_name` and save
2. visit http://vote-gov.lndo.site/register/alaska and confirm the page title is updated
3. visit http://vote-gov.lndo.site/tl/register/alaska and confirm the page title is showing in Tagalog

## Checklist for the Developer




## Checklist for the Peer Reviewers
